README.md

```diff
@@ -73,52 +73,47 @@ Here is a list of reasons how we improved on zap with slog.
    - We wanted an API that only accepted the equivalent of [zap.Any](https://godoc.org/go.uber.org/zap#Any) for every field.
      This is [slog.F](https://godoc.org/cdr.dev/slog#F).
 
-1. `slog` uses a very
+1. `sloggers/sloghuman` uses a very human readable format
 
-   - Lack of appropriate colors for different levels and fields
-     - slog colors distinct parts of each line to make it easier to scan logs. Even the JSON that represents
-       the fields in each log is syntax highlighted so that is very easy to scan. See the screenshot above.
-   - zap logs multiline fields and errors stack traces as JSON strings which made them unreadable in a terminal.
-     - slog automatically prints one multiline field after the log to make errors and such much easier to read.
-       slog also automatically prints a Go 1.13 error chain as an array. See screenshot above.
+   - slog colors distinct parts of each line to make it easier to scan logs. Even the JSON that represents
+     the fields in each log is syntax highlighted so that is very easy to scan. See the screenshot above.
+     - zap lacks appropriate colors for different levels and fields
+   - slog automatically prints one multiline field after the log to make errors and such much easier to read.
+     slog also automatically prints a Go 1.13 error chain as an array. See screenshot above.
+     - zap logs multiline fields and errors stack traces as JSON strings which made them unreadable in a terminal.
 
-1. Lack of [context.Context](https://blog.golang.org/context) support.
+1. Full [context.Context](https://blog.golang.org/context) support
+   `slog` lets you set fields in a `context.Context` such that any log with the context prints those fields.
```

Comment:
This is a bit weird because they end up on the same line.

Should the second line be a bullet point?

Reply:
Will fix.